firefox-ios/Shared/Prefs.swift

```diff
@@ -199,6 +199,9 @@ public struct PrefsKeys {
     // Represents whether or not the user has seen the photon main menu once, at least.
     public static let PhotonMainMenuShown = "PhotonMainMenuShown"
 
+    // Most recently created or saved to bookmark folder that we should save all future bookmarks to
+    public static let BookmarkSaveToFolder = "BookmarkSaveToFolder"
```

Comment:
Could this be something like `LastSavedBookmarkFolder` ?

Reply:
`MostRecentBookmarkFolder` or `RecentBookmarkFolder` Maybe?

The comment might read a bit better if you split it into one more sentence.
`// The bookmark folder that was most recently created or saved to by the user. `
`// Used to indicate where we should save the next bookmark by default.`